hr_operating_unit/models/hr_operating_unit.py

```diff
@@ -0,0 +1,15 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrOperatingUnit(models.Model):
```

Comment:
Non blocking. But I think it is more proper.
```suggestion
class OperatingUnit(models.Model):
```